set.mm

```diff
@@ -59314,12 +59316,11 @@ in the range of the function (the implication "to the right" is always
   $}
 
   ${
-    $d A t $.  $d C t $.
-    mptex2.1 $e |- ( ph -> ( t e. A |-> B ) : A --> C ) $.
-    $( If a class given in maps-to notation is a function with domain and
-       codomain, then its values are sets.  (Contributed by Glauco Siliprandi,
-       11-Dec-2019.) $)
-    mptex2 $p |- ( ( ph /\ t e. A ) -> B e. C ) $=
+    $d A x $.  $d C x $.
+    fvmptelrn.1 $e |- ( ph -> ( x e. A |-> B ) : A --> C ) $.
+    $( The value of a function at a point of its domain belongs to its
+       codomain.  (Contributed by Mario Carneiro, 29-Dec-2016.) $)
+    fvmptelrn $p |- ( ( ph /\ x e. A ) -> B e. C ) $=
       ( wcel cmpt wf wral eqid fmpt sylibr r19.21bi ) ADEGZBCACEBCDHZIOBCJFBCED
```

Comment:
"rn" is often used as label fragment when "codomain" is meant, see ~ffvelrn and ~ffvelrnda. So we would have to chane this everywhere consistently. For now, we can keep this name.